llvm/test/CodeGen/PowerPC/aix-alias-alignment.ll

```diff
@@ -62,7 +62,7 @@ define void @foo(i32 %a1, i32 %a2, i32 %a3) {
 
 ; SYM:      SYMBOL TABLE:
 ; SYM-NEXT: 00000000      df *DEBUG*	00000000 <stdin>
-; SYM-NEXT: 00000000 l       .text	00000029 
+; SYM-NEXT: 00000000 l     F .text	00000029 
```

Comment:
This seems wrong. This symbol is not a function. If you code requires this testcase change, then I don't think the code is right. You need to see if a label is at the same address, and if so, the csect is not a function.